### PR TITLE
NDS conversion

### PR DESCRIPTION
**Pull request by cdemetriadis**
First version of the NDS Sketch Plugin

### CODE REVIEW

**Review by kerrmarin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# sketch-switch-theme-library-plugin
\ No newline at end of file
+# NDS Sketch Plugin
+
+### The Natural Design System
+
+### How the plugin works
+
```
[the remark this answers is not in the document]
Comment (cdemetriadis) [reply README.md:6]: Yes, just headline notes for what I need to add now.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "@skpm/builder": "^0.7.4",
+    "@skpm/extract-loader": "^2.0.2",
+    "css-loader": "^3.2.0",
+    "html-loader": "^0.5.5",
+    "jest": "^25.2.4"
+  },
+  "dependencies": {
+    "babel-jest": "^25.2.4",
+    "babel-preset-env": "^1.7.0",
+    "sketch-module-web-view": "^3.1.4",
```
[the remark this answers is not in the document]
Comment (cdemetriadis) [reply package.json:38]: True

File: package.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@skpm/builder": "^0.7.4",
+    "@skpm/extract-loader": "^2.0.2",
+    "css-loader": "^3.2.0",
+    "html-loader": "^0.5.5",
+    "jest": "^25.2.4"
+  },
+  "dependencies": {
+    "babel-jest": "^25.2.4",
```
[the remark this answers is not in the document]
Comment (cdemetriadis) [reply package.json:36]: OK

File: src/lib/context.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    }
+    toString() {
+        return this._arr.join("/")
+    }
+    append(str) {
+        return new VariableSizeContext(this.toString()+"/"+str)
+    }
+    appendLast(str) {
+        const arr = this._arrayFromString(str)
+        return this.append(arr[arr.length-1])
```
[the remark this answers is not in the document]
Comment (cdemetriadis) [reply src/lib/context.js:20]: Changed to this:

```
if (arr.length > 0) {
  return this.append(arr[arr.length-1])
} else {
  return this
}
```

**Pull request merged**